lib/ae_mdw/db/mutations/stats_mutation.ex

```diff
@@ -202,6 +202,8 @@ defmodule AeMdw.Db.StatsMutation do
 
     token_supply_delta = AeMdw.Node.token_supply_delta(height - 1)
     auctions_expired = get(state, :auctions_expired, 0)
+    new_oracles_expired = get(state, :new_oracles_expired, 0)
+    old_oracles_registered = get(state, :old_oracles_registered, 0)
```

Comment:
Can we keep the notation used before (without the `old/new` prefix)?

Reply:
Sorry, the description of the problem was missing. These are for:
- `new_oracles_expired`: avoids double counting inactive oracles by checking if it was the first time the oracle was created (without previous).
- `old_oracles_registered`: decrements inactive ones that are reactivated.

Reply:
I hadn't realized oracles are mapped 1-to-1 to the address of the creator (unlike contracts or channels), meaning that the owner can create multiple oracles (not simultaneously, I guess?). Thank you for clearing it up